src/main/java/com/google/sps/lib/Job.java

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.lib;
+
+import java.lang.Exception;
+import com.google.gson.Gson;
+import javax.servlet.http.HttpServletRequest;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.util.UUID;
+import com.google.sps.lib.ErrorResponse;
+
+/** 
+ * This class contains essential functions for a new Job.
+ */  
+public class Job {
+
+  private static final String ENTITY_JOBID__PROPERTY_NAME = "jobId";
```

Comment:
Please keep all entity names in TitleCase.

```suggestion
  private static final String ENTITY_JOBID__PROPERTY_NAME = "JobId";
```

Reply:
Also, why are there two underscores in name?